application/templates/application/documentation.html

```diff
@@ -0,0 +1,19 @@
+{% extends "application/global-html.html" %}
+{% block body %}
+    <div style="text-align: center;">
+        <div class="container mx-auto mt-4">
+            <div class="row">
+                <div class="col-md">
+                        <button class="btn btn-primary mt-4 btn-block"
+                                onclick="window.location.href={% url 'logout-view' %}">User Documentation
```

Comment:
This should link to the github documentation page.
User: 'https://github.com/Brick7Face/psychiatric-guide-app/blob/master/User_Documentation.md'
Developer: 'https://github.com/Brick7Face/psychiatric-guide-app/blob/master/README.md'